pages/posts/github-case-study.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "How JSON Schema Was an Obvious Choice at GitHub"
+date: 2023-07-31
+type: Case Study
+cover: /img/posts/2023/github-case-study/github-mark.svg
```

Comment:
This SVG needs shrinking probably, I guess to some extent it too was a placeholder.

Reply:
Prefer webp. You can download the cli converter from here: https://developers.google.com/speed/webp

Yeah, I think it is a placeholder. Will need something else.

Reply:
Surely webp is preferred over png/jpg/etc. but not for svg? Though if/when we have an idea for a replacement probably it won't matter.

Reply:
You know, I missed that it was svg.
But anwyay, I expect it will be replaced. I... didn't realise what it was when I made that comment anyway.